### DISCUSSION TITLE
Welcome to stable-audio-tools Discussions!

### DISCUSSION THREAD

**Discussion by zqevans** [Stability-AI/stable-audio-tools#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 0417keito** [comment]
Thank you for the great discussion. This is a very beneficial place for me.

I am a second year university student at a Japanese university. I am currently studying music and voice.
Neither Suno's chirp nor more recently Riffusion's model of singing voice generation with music is open source, so I am studying it because it would be interesting if I could open source it.

I have come up with the following ideas.
What about the following architecture for generating singing voices with music?
The methodology mainly followed SPEAR-TTS.
Sorry for the handwriting.
![image](https://github.com/Stability-AI/stable-audio-tools/assets/117526884/07cf4235-4c8a-49cd-85b8-f0ed7fd0a7c2)

I have written the methodology for this approach and the code, although it is not completed, here.
https://github.com/0417keito/UTAUTAI

One drawback of this arch is that it uses AR and diffusion model, so it is slow to generate.

**Comment by Ssojux2** [comment]
Hi from South Korea.

We're Rubato Lab in MODULABS.

**Comment by danishbelim** [comment]
How does the Stable-audio-tool facilitate the synchronization and integration of tabular data with audio input for enhanced data analysis and [accessibility](https://composio.dev/)?

**Reply by danishbelim** [reply]
you can check this here https://blog.composio.dev/gpt-4-function-calling-example/
